cmd/finch/virtual_machine_status_test.go

```diff
@@ -0,0 +1,17 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewStatusVMCommand(t *testing.T) {
```

Comment:
I think there can be at least 5 unit test scenarioes, including 3 valid statuses and 2 invalid statuses. You can use "make coverage" to self-check the unit test coverage.

Reply:
Trying to figure out the mocking in the next days and try to implement the test scenarios.